src/screens/TopicPageScreen/index.js

```diff
@@ -53,7 +54,9 @@ const TopicPageScreen = (props) => {
 
   const [offset, setOffset] = React.useState(0);
   const [client] = React.useContext(Context).client;
+  const [user] = React.useContext(Context).profile;
   const refBlockComponent = React.useRef();
+  const bottomSheetFollowRef = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `user` context and `bottomSheetFollowRef` reference are important changes. The `user` context is likely used to access user profile data, while `bottomSheetFollowRef` is used to control the visibility of the `BottomSheetFollow` component.
```diff
+ const [user] = React.useContext(Context).profile;
+ const bottomSheetFollowRef = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->